n3fit/src/n3fit/scripts/evolven3fit_new.py

```diff
@@ -0,0 +1,376 @@
+"""
+This module contains the CLI for evolven3fit_new
+"""
+import logging
+import pathlib
+from argparse import ArgumentParser
+import sys
+import numpy as np
+
+from evolven3fit_new import eko_utils, evolve, utils
+
+log = logging.getLogger(__name__)
+
+
+def main():
+    parser = ArgumentParser(
+        description="evolven3fit_new - a script with tools to evolve PDF fits"
+    )
+    subparsers = parser.add_subparsers(title="actions", dest="actions")
+    evolvefit_parser = subparsers.add_parser(
+        "evolve",
+        help="Evolves the fitted PDFs. The q_grid starts at the Q0 given by the theory but the last point is q_fin and its number of points can be specified by q_points. If a path is given for the dump option, the eko will be dumped in that path after the computation. If a path is given for the load option, the eko to be used for the evolution will be loaded from that path. The two options are mutually exclusive.",
+    )
+    eko_parser = subparsers.add_parser(
+        "produce_eko",
+        help="Produce the eko for the specified theory. The q_grid starts at the Q0 given by the theory but the last point is q_fin and its number of points can be specified by q_points. The x_grid starts at x_grid_ini and ends at 1.0 and contains the provided number of points. The eko will be dumped in the provided path.",
+    )
+    evolvefit_parser.add_argument(
+        "configuration_folder",
+        help="Path to the folder containing the (pre-DGLAP) fit result",
+    )
+    parser.add_argument(
+        "-q",
+        "--q_fin",
```

Comment:
Do not use underscores in arguments, it's not natural (and more annoying to type...). In any case, `argparse` will convert the variable to `q_fin`, since dashes are not valid in identifiers
```suggestion
        "--q-fin",
```